test/distributed/org/apache/cassandra/distributed/test/accord/journal/StatefulJournalRestartTest.java

```diff
@@ -0,0 +1,136 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.cassandra.distributed.test.accord.journal;
+
+import java.io.IOException;
+import java.time.Duration;
+
+import org.junit.Ignore;
+import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import accord.local.durability.DurabilityService;
+import accord.primitives.Ranges;
+import accord.primitives.Timestamp;
+import accord.utils.Property;
+import accord.utils.Property.SimpleCommand;
+import accord.utils.async.AsyncChains;
+import org.apache.cassandra.db.Keyspace;
+import org.apache.cassandra.distributed.Cluster;
+import org.apache.cassandra.distributed.api.IInvokableInstance;
+import org.apache.cassandra.distributed.shared.ClusterUtils;
+import org.apache.cassandra.distributed.test.TestBaseImpl;
+import org.apache.cassandra.schema.TableMetadata;
+import org.apache.cassandra.service.accord.AccordService;
+import org.apache.cassandra.service.accord.TokenRange;
+import org.apache.cassandra.service.consensus.TransactionalMode;
+
+import static accord.utils.Property.commands;
+import static accord.utils.Property.stateful;
+import static org.apache.cassandra.schema.SchemaConstants.ACCORD_KEYSPACE_NAME;
+import static org.apache.cassandra.service.accord.AccordKeyspace.JOURNAL;
+
+/**
+ * There are 2 errors blocking this test from being run
+ * <p>
+ * <pre>
+ * INFO  [node1_AccordExecutor[0,13]] 2025-05-27 16:02:51,792 SubstituteLogger.java:169 - ERROR 23:02:51 Uncaught accord exception
```

Comment:
we publish `AccordService` *before* journal reply, so logically this shouldn't be "caused" by the reply issue below... if its caused by journal reply then wouldn't this be a race condition issue then? We are taking on messages *before* we can handle them?

Reply:
also this is a single node test, so these messages are being sent by *self*